doc/api/http.md

```diff
@@ -1288,6 +1288,14 @@ the server, then sockets are destroyed when they time out. If a handler is
 assigned to the request, the response, or the server's `'timeout'` events,
 timed out sockets must be handled explicitly.
 
+### response.aborted
+<!-- YAML
+added: v10.0.0
```

Comment:
This should be `added: REPLACEME`. The actual version will then be substituted by the releaser when the change first lands in a release.